src/pages/components/AllCompensations/AllCompensationsContainer.tsx

```diff
@@ -43,5 +44,7 @@ export const AllCompensationsContainer = observer(() => {
     >(
       `${LINK_TO_COMPENSATIONS_SERVICE}${compensationId}/soft-delete`,
     )
+
+    allCompensationsState.updateStatus(true)
```

Comment:
In the workshop it was the page that was calling state method to trigger reload. Here it could be something like onCompensationDeleted callback call.